distributions/stacks/openshift-managed/OWNERS

```diff
@@ -0,0 +1,4 @@
+approvers:
+- nakfour
+- vpavlin
+- crobby
```

Comment:
Not sure why we are adding a new stack called "openshift-managed", we should not have multiple stacks for Openshift

Reply:
Created a new stack due to following reasons:
1. In future we could include OpenShift managed components like - Openshift Serverless operator, cert-manager operator.
2. Having a separate stack can enable users to have kubeflow applications with upstream Istio on OpenShift if they want to.
3.  The kustomization files in `openshift-managed` stack differ from the `openshift` stack , where applications running with ServiceMesh have additional label- https://github.com/VaishnaviHire/manifests/blob/test_sm_istio/distributions/stacks/openshift-managed/add_label.yaml#L1

We can have one stack if we want to require users to only use ServiceMesh on OpenShift.